README.md

```diff
@@ -210,7 +210,7 @@ pip install -r requirements.lock
 # Generate protos
 python -m grpc_tools.protoc --proto_path=. \
   --python_out=. --grpc_python_out=. --pyi_out=. \
-  protos/grpc/testing/*.proto
+  protos/grpc/testing/*.proto protos/grpc/testing/**/*.proto 
```

Comment:
```suggestion
  $(find protos/grpc/testing/ -type f -name '*.proto')
```
I don't like bash globbing - it only gets results people expect when `shopt -s globstar` is enabled. Let's replace it with `find` instead.

Reply:
Replaced with the dir name. This is readme, need to make it easy to understand and follow

Reply:
Ok, looks good.